infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -469,8 +497,14 @@ func (b *TFBlueprintTest) DefaultInit(assert *assert.Assertions) {
 
 // Vet runs TF plan, TF show, and gcloud terraform vet on a blueprint.
 func (b *TFBlueprintTest) Vet(assert *assert.Assertions) {
+	vetTempDir, err := os.MkdirTemp(os.TempDir(), "btp")
+	if err != nil {
+		b.t.Fatalf("Temp directory <%s> could not created: %v", vetTempDir, err)
```

Comment:
nit: %q will quote the string
```suggestion
		b.t.Fatalf("Temp directory %q could not created: %v", vetTempDir, err)
```

Reply:
Thanks - I always forget about `%q` after working in other languages. :)